### DISCUSSION TITLE
Disable background sync

### DISCUSSION THREAD

**Discussion by metbril** [MobiusSync/MobiusSync#116] [Q&A] [answered]
I currently don't need syncing my iPhone and want to disable background sync. I have paused the device and the remote computer. However I still get notifications about syncing. 

How can I (temporarily) completely disable background syncing without uninstalling the app?

**Comment by SimonPickup** [comment]
[Image.png]

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Robert ***@***.***>
Sent: Friday, May 19, 2023 7:48:26 AM
To: MobiusSync/MobiusSync ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MobiusSync/MobiusSync] Disable background sync (Discussion #116)


I currently don't need syncing my iPhone and want to disable background sync. I have paused the device and the remote computer. However I still get notifications about syncing.

How can I (temporarily) completely disable background syncing without uninstalling the app?

**Reply by metbril** [reply]
That image didn't get through.

**Comment by SimonPickup** [comment, marked as answer]
In Settings/iOS, you can set the timeouts to 0:

![mobius](https://github.com/MobiusSync/MobiusSync/assets/18005537/8bc0d7f4-280b-498f-9a71-b8c61e61a52d)

**Reply by metbril** [reply]
Thanks. That's a well hidden setting.

**Reply by metbril** [reply]
Well, that didn't last long... I still get notifications. So something is still running.

**Comment by SimonPickup** [comment]
You can also disable "Background App Refresh" in iOS settings for the app.
